x-pack/plugins/infra/public/pages/metrics/hosts/hooks/use_alerts_query.ts

```diff
@@ -65,22 +65,21 @@ const createAlertsEsQuery = ({
   hostNodes: SnapshotNode[];
   status?: AlertStatus;
 }): AlertsEsQuery => {
-  const alertStatusQuery = createAlertStatusQuery(status);
+  const alertStatusFilter = createAlertStatusFilter(status);
 
   const dateFilter = createDateFilter(dateRange);
   const hostsFilter = createHostsFilter(hostNodes);
 
-  const queries = alertStatusQuery ? [alertStatusQuery] : [];
-  const filters = [hostsFilter, dateFilter].filter(Boolean) as Filter[];
+  const filters = [alertStatusFilter, dateFilter, hostsFilter].filter(Boolean) as Filter[];
 
-  return buildEsQuery(undefined, queries, filters);
+  return buildEsQuery(undefined, [], filters);
```

Comment:
Why aren't we passing a data view here? Is it because we have more control over the filters that are passed to it?

Reply:
As done in the `observability` plugin, the data view was not passed because we need more control over those filters yeah! This way we are just taking care of passing the list of hosts, the data range and the alert status, without the need to retrieve the alerts data view.